server_apps/data_transfer/PUBMED/updateExistingPubs.sql

```diff
@@ -0,0 +1,105 @@
+begin work;
+
+create temp table tmp_pubs (
+  pmcid text,
+  mid text,
+  pmid integer,
+  keywords text,
+  title text,
+  pages text,
+  abstract text,
+  authors text,
+  numAuthors int,
+  year text,
+  month text,
+  day text,
+  issn text,
+  volume text,
+  issue text,
+  journaltitle text,
+  iso text,
+  status text,
+  pubType text);
+
+\copy tmp_pubs from 'parsePubs.log';
+
+delete from tmp_pubs
+where (authors = 'none' or authors is null)
+      and numAuthors = '0';
+
+create temp table tmp_new_pubs (
+  pmcid text,
+  mid text,
+  zdb_id text,
+  pmid integer,
+  keywords text,
+  title text,
+  pages text,
+  abstract text,
+  authors text,
+  year text,
+  month text,
+  day text,
+  issn text,
+  volume text,
+  issue text,
+  journaltitle text,
+  iso text,
+  status text,
+  journal_zdb_id text,
+  pubType text
+);
+
+insert into tmp_new_pubs
```

Comment:
Do you need to drop this table at the end of this script?

Reply:
As a temp table, these should get dropped automatically.